### PR TITLE
Added the Phantom role

### PR DESCRIPTION
**Pull request by FoppyOmega**
The Phantom role is an imposter that can cloak itself for x seconds turning completely invisible. During the cloak period its speed is cut in half.

### CODE REVIEW

File: Source Code/GameOptionsPatch.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ private static void Postfix(ref string __result)
             var hudString = stringBuilder.ToString();
 
             int defaultSettingsLines = 19;
-            int roleSettingsLines = 19 + 23;
+            int roleSettingsLines = 19 + 24;
```
Comment (FoppyOmega) [inline Source Code/GameOptionsPatch.cs:53]: These values may not be correct anymore now that vampire was also added.

**Comment by Eisbison** [comment]
I'm not sure if we want to add this role to the mod, because it already exists in other mods and we're trying to add some unique stuff, but good work there 😉 👍

**Comment by FoppyOmega** [comment]
> I'm not sure if we want to add this role to the mod, because it already exists in other mods and we're trying to add some unique stuff, but good work there 😉 👍

Be 100% honest, and don't just try to be nice and brush me off; if I were to create a unique and fun role, would you add it into your mod? If you want to keep the coding to yourself and only add in your own roles then I'm totally cool with that.
I won't waste any more time if you prefer to keep it in house. I don't buy the "unique" angle because most of your roles are taken from other places. Phantom is at least unique with the speed decrease to help balance the role. Anyway thank you for all your efforts! 👍👍 You've got my support if you'd like it.

**Comment by Eisbison** [comment]
Yes I would add it, e.g. Thunderstorm created the Jackal/Sidekick and we added that and now he's part of the dev team. And I don't want to keep the coding to myself, I'm more than happy to load off some of the work... things start to get stressful as the project keeps growing. Regarding the unique thing, yes you're right, but we at least tried to rework all the stuff and for each new role we tried to add something, such that it's balanced and a bit unique. Maybe let's have a chat on Discord about the role, I think we can turn it into a role we should add 👍

**Pull request closed without merging**